src/components/sbb-dialog/sbb-dialog.scss

```diff
@@ -0,0 +1,256 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-dialog-padding-inline: var(--sbb-spacing-fixed-5x);
+  --sbb-dialog-color: var(--sbb-color-black-default);
+  --sbb-dialog-background-color: var(--sbb-color-white-default);
+  --sbb-dialog-max-width: 100%;
+  --sbb-dialog-max-height: 100%;
+  --sbb-dialog-border-radius: var(--sbb-border-radius-8x);
+  --sbb-dialog-padding-block: var(--sbb-spacing-responsive-s);
+  --sbb-dialog-animation-duration: var(--sbb-animation-duration-6x);
+  --sbb-dialog-animation-easing: ease;
+  --sbb-dialog-backdrop-color: transparent;
+  --sbb-dialog-header-padding-block: var(--sbb-spacing-responsive-s) 0;
+  --sbb-dialog-footer-border: var(--sbb-border-width-1x) solid var(--sbb-color-cloud-default);
+  --sbb-scrollbar-color: var(--sbb-color-black-alpha-30);
+  --sbb-scrollbar-color-hover: var(--sbb-color-black-alpha-60);
+
+  position: fixed;
+  pointer-events: none;
+  display: flex;
+  align-items: center;
+  inset: 0;
+  z-index: 10;
```

Comment:
I'm not sure why this styles are needed? so the sbb-dialog, even if not displayed, always overlaps all the content on the page?